routing/routing_tests/index_graph_test.cpp

```diff
@@ -37,8 +37,11 @@ using namespace routing_test;
 
 using TestEdge = TestIndexGraphTopology::Edge;
 
+double constexpr kKMPH2MPS = 1000.0 / (60 * 60);
```

Comment:
It's the 4th time this constant is going to be defined in routing/
Maybe we should move it to somewhere like routing_helpers.

Reply:
done